galaxy.xsd

```diff
@@ -2099,13 +2115,12 @@ code file wraps a metadata collection to return metadata parameters wrapped
 according to the Metadata spec.
 
 ```xml
-<command>
```

Comment:
I tried the nested CDATA thing somewhere else and it didn't work for me. I can try again though.